pwa/static/themes/rotterdam.css

```diff
@@ -168,6 +168,12 @@
   --web-app-component-image-max-width: 300px;
   --web-app-component-rating-indicator-max-height: 160px;
 
+  --web-app-ornaziation-detail-logo-height: 300px;
+  --web-app-ornaziation-detail-logo-width: 350px;
+
+  /* This token is TEMPORARY and only for DEV & DEMO purposes */
+  --web-app-ornaziation-detail-old-logo-height: 300px;
```

Comment:
I think we should just remove this and keep to the dynamic template, instead of starting to add these kind of tokens.

If after reading all feedback, I'm missing something, and we do decide to keep that old implementation: I'd opt to _not_ create a design token for it (it's a temporary hardcoded thing, no need to make it dynamic).

Another note: we didn't add this to `theme.css.example`, which we should have (if we were to forget removing the functionality and token altogether, _and then_ create a new theme, this would break).